PlaidLink.js

```diff
@@ -130,7 +140,18 @@ PlaidLink.propTypes = {
   // Specify an existing user's public token to launch Link in update mode.
   // This will cause Link to open directly to the authentication step for
   // that user's institution.
-  token: PropTypes.string,
+  // Pass an item_add_token to launch Link in regular mode without a public_key.
+  // Either publicKey or token is required.
+  token: props => {
+    if (!props.publicKey && !props.token) {
```

Comment:
I think we only need this validation on one of these fields since they check for the same thing. We could include the validation for the publicKey but leave token as `token: PropTypes.string`

Reply:
Good call, will update -- thanks!